src/components/Table/TableHeaderCell.vue

```diff
@@ -5,13 +5,15 @@
         :sticky-right="stickyRight"
         :sortable="column.sortable"
         @click="onClick">
-        <span class="table-header-label">
-            {{ column.name }}
-        </span>
-        <template v-if="column.sortable">
-            <ChevronDownIcon v-if="sorted && sortedDesc" size="small"/>
-            <ChevronUpIcon v-if="sorted && !sortedDesc" size="small"/>
-        </template>
+        <slot :name="headerSlotName(column.id)">
```

Comment:
I think we should pass all the column parameters here (`sortable`, `sorted`, `sortedDesc`) in case someone would like to use it as actual sortable header, and not checkbox.